pkg/ocm/helpers.go

```diff
@@ -684,23 +681,13 @@ func ValidateHostedClusterSubnets(awsClient aws.Client, isPrivate bool, subnetID
 		return 0, vpcSubnetsErr
 	}
 
-	var subnets []*ec2.Subnet
-	for _, subnet := range vpcSubnets {
-		for _, subnetId := range subnetIDs {
```

Comment:
It seems like we are filtering through the user choices before calling `FilterVPCsPrivateSubnets`, why are we removing this part?

Reply:
mistake on my part, I thought it was just an uneccessary conversion to a pointer